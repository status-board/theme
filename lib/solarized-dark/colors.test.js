"use strict";

var _colors = _interopRequireDefault(require("./colors"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Colors should match snapshot', function () {
  expect(_colors["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9zb2xhcml6ZWQtZGFyay9jb2xvcnMudGVzdC50cyJdLCJuYW1lcyI6WyJ0ZXN0IiwiZXhwZWN0IiwiY29sb3JzIiwidG9NYXRjaFNuYXBzaG90Il0sIm1hcHBpbmdzIjoiOztBQUFBOzs7O0FBRUFBLElBQUksQ0FBQyw4QkFBRCxFQUFpQyxZQUFZO0FBQzdDQyxFQUFBQSxNQUFNLENBQUNDLGtCQUFELENBQU4sQ0FBZUMsZUFBZjtBQUNILENBRkcsQ0FBSiIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBjb2xvcnMgZnJvbSAnLi9jb2xvcnMnO1xuXG50ZXN0KCdDb2xvcnMgc2hvdWxkIG1hdGNoIHNuYXBzaG90JywgKCk6IHZvaWQgPT4ge1xuICAgIGV4cGVjdChjb2xvcnMpLnRvTWF0Y2hTbmFwc2hvdCgpO1xufSk7XG4iXX0=