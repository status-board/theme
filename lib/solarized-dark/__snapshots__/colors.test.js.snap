// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors should match snapshot 1`] = `
Object {
  "background": "#073642",
  "failure": "#dc322f",
  "icon": "#0badc2",
  "success": "#859900",
  "text": "#90a2a4",
  "textHighlight": "#eee8d5",
  "unknown": "#495b71",
  "warning": "#b58900",
}
`;
